Makefile

```diff
@@ -1,3 +1,4 @@
+LOCALE_PREFIX ?= /usr
```

Comment:
Does `LOCALE_PREFIX=$(PREFIX)/share/locale` not work? I don't like how this is inconsistent with to `MANPREFIX`

Reply:
The reason I had to make this different is that the locales have to installed at the `/usr/share/locale` prefix instead of the `/usr/local/share/locale` prefix which would result from `$(PREFIX)/share/locale` (which maps to `MANPREFIX`). This is predefined because of `export TEXTDOMAINDIR=/usr/share/locale` in `downgrade`.

I mofieid `LOCALE_PREFIX` to be slightly more similar to `MANPREFIX`.

Reply:
WDYT?

Reply:
> This is predefined because of export `TEXTDOMAINDIR=/usr/share/locale` in downgrade.

Ah crap, that's a pre-existing packaging bug then. If one were to `make install PREFIX=/opt`, _everything_ should live in `/opt`, and `downgrade` should use `TEXTDOMAINDIR=/opt/share/locale`. So really, my `Makefile` suggestion is correct, but the hard-coded `TEXTDOMAINDIR` is not right.

I see three options:

- Ignore this (for now), and keep what you're doing
- Do as I suggested for `LOCALE_PREFIX`, and make it work by:

  Making `downgrade` a `.in` file, with an interpolation for `@PREFIX@` in the `TEXTDOMAINDIR` definition. Pre-process `downgrade.in` to `downgrade`, filling in `PREFIX` along the way.

- Do as I suggested for `LOCALE_PREFIX`, and make it work by:

  Writing a file as part of `make install`, such as `/etc/profile.d/dowgrade.sh`, which includes the `export TEXTDOMAINDIR` with a `PREFIX`-prefixed value. `souce` this `.env` file from the `downgrade` script.

Do you see any other options? What do you think of those 3?

Reply:
I understand now. Suggestion 2 with `downgrade.in` seems the least intrusive for me and will probably result in the smallest diff. I will commit a suggestion for this and get your feedback.

Reply:
I am not so familiar with `configure` and `make` routines, so I just committed the simplest solution I could think of. Converted `downgrade` to `downgrade.in` with a special variable `@LOCALE_PREFIX@` which needs to be substituted before installation. Updated `Makefile` to reflect this. The installation process for the `/opt` prefix would be:

```
$ PREFIX=/opt make config install install.po
```